app/views/likes/create.js.erb

```diff
@@ -0,0 +1,3 @@
+$("#likes_<%= @recipe.id %>").html('<%= @recipe.likes.count %>&nbsp;');
```

Comment:
even the `@recipe.likes.count` is cached when its called twice, it's best practice to store this in a var so it won't be calculated again.

Reply:
Make sence, thanks for the tip!
Done!

Reply:
done!